gems/manageiq-providers-amazon/lib/manageiq-providers-amazon.rb

```diff
@@ -0,0 +1,5 @@
+module ManageIQ::Providers::Amazon
+  class Engine < ::Rails::Engine
+    isolate_namespace ManageIQ::Providers::Amazon
```

Comment:
@matthewd this is probably the reason, why this move was so smooth so far.
what do you think about making this an engine?
we could try to setup all autoloading and stuff by hand, but this seems to work out of the box...
and I have a gut feeling, that we might need more of the railtie features?! but that might be YAGNI